src/Worker/AbstractWorker.php

```diff
@@ -73,13 +87,27 @@ protected function work($timeout = null)
             return;
         }
 
+        $jobsdone   = 0;
+        $break      = false;
+        $lastActive = time();
         while (true) {
             $job = $this->adapter->pickTask($timeout);
 
+            /**
```

Comment:
does it work, wasnt it supposed to go at bottom of function to prevent next loop?